css/styles.css

```diff
@@ -659,3 +666,479 @@ p#modal-close {
+    display: inline-block;
+  }
+
+  .categories li {
+    color: #6070ff;
+  }
+
+  .card-body h2 {
+    font-size: 4rem;
+  }
+
+  .card-body p {
+    font-size: 1.6rem;
+  }
+
+  .small-para {
+    display: none;
+  }
+
+  .text-light {
+    display: none;
+  }
+
+  .large-list {
+    display: inline-block;
+  }
+
+  .small-heading {
+    display: inline-block;
+  }
+
+  .text-normal {
+    display: none;
+  }
+
+  /* About Section */
+
+  .about-page {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    max-width: 100%;
+    align-items: center;
+    height: 644px;
+    background: var(--white);
+    width: 110%;
+  }
+
+  .about-header {
+    font-size: 4rem;
+    margin-top: 5rem;
+  }
+
+  .language-section {
+    padding-left: 59px;
+    max-width: 56rem;
+    margin: 516px auto;
+    margin-top: -5rem;
+    top: 35%;
+    left: 167px;
+  }
+
+  .about-container {
+    padding-left: 59px;
+    max-width: 56rem;
+    margin: 633px auto;
+    margin-top: 5rem;
+    top: 35%;
+    left: 167px;
+  }
+
+  .about-page p {
+    color: var(--color-text);
+    line-height: 2.4rem;
+    font-size: 1.2rem;
+    margin-top: 0.5rem;
+    left: 1px;
+    padding-right: 100px;
+  }
+
+  .about-button {
+    padding: 1.5rem 2.2rem;
+    margin-top: 3rem;
+    border-radius: 10px;
+    border: 1px solid #396df2;
+    color: #396df2;
+    font-size: 1.7rem;
+    background-color: #fff;
+  }
+
+  .about-button:hover {
+    background-color: #6070ff;
+    color: var(--white);
+    transition: all 1000ms ease-in-out;
+  }
+
+  .about-button:active {
+    background-color: #2230d2;
+    color: var(--white);
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+    padding-right: 115px;
+  }
+
+  .languages li {
+    width: 122px;
+    height: 120px;
+    background-color: #f7f7f9;
+    margin-top: 0.8rem;
+    gap: 1.5rem;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  /* Contact Section */
+
+  .contact-sectionbro {
+    height: 120vh;
+    background: #6070ff url(../images/contactform.svg);
+    object-fit: cover;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    border-top-left-radius: 75px;
+    color: var(--white);
+    width: 100%;
+    top: -70px;
+    margin-top: auto;
+    text-align: center;
+    position: relative;
+  }
+
+  .contact-btn {
+    margin-top: 1.2rem;
+  }
+
+  .contact-paragraph {
+    font-size: small;
+    width: 379px;
+  }
+
+  .contact-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin-bottom: 6rem;
+    height: 800px;
+    max-height: 90%;
+    max-width: 100%;
+  }
+
+  .flex-item1 {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .form-inputs {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 447px;
+  }
```

Comment:
- [ ] Kindly increase the width of the form input to make sure that this form 😄 , it will make your design stick to the Figma design 👍 

![Capture 1](https://user-images.githubusercontent.com/80458199/229279295-630da10d-ae49-4b5f-b02e-7454f0c75b42.PNG)